### PR TITLE
Explicitly inject dependencies and fix issues

### PR DESCRIPTION
**Pull request by jameesjohn**
This PR fixes dependency injection issue which occurs upon minification and also fixes issues with `strict mode`

### CODE REVIEW

File: src/ui-codemirror.js [hunk 1]
```diff
@@ -148,3 +147,5 @@ function uiCodemirrorDirective($timeout, uiCodemirrorConfig) {
   }
 
 }
+
+uiCodemirrorDirective.$inject = [ '$timeout', 'uiCodemirrorConfig' ];
```
Comment (seanlip) [inline src/ui-codemirror.js]: Add newline at EOL.

Comment (jameesjohn) [reply src/ui-codemirror.js]: done

Comment (jameesjohn) [reply src/ui-codemirror.js]: Done

**Comment by seanlip** [comment]
Thanks, LGTM.

**Pull request merged**